lib/index.ts

```diff
@@ -28,3 +28,6 @@ export type { PrismaOtelTracingPluginConfig } from './plugins/opentelemetry/pris
 
 export { publicHealthcheckPlugin } from './plugins/publicHealthcheckPlugin'
 export type { PublicHealthcheckPluginOptions, HealthCheck } from './plugins/publicHealthcheckPlugin'
+
+export { amplitudePlugin, amplitudeTrack } from './plugins/amplitudePlugin'
+export type { AmplitudeConfig, createApiTrackingEventFn } from './plugins/amplitudePlugin'
```

Comment:
🟢 You can inline type-only imports:

```ts
export {
  amplitudePlugin,
  amplitudeTrack,
  type AmplitudeConfig,
  type createApiTrackingEventFn,
} from './plugins/amplitudePlugin'
```

Reply:
I didn't know it! thanks 🙇, I will apply it and have it in mind for the future 😄

Reply:
It's a newish feature in TypeScript :-)